### ISSUE TITLE
After upgrade 3.6.1 an error on initialization is thrown

### ISSUE THREAD

**Issue by arekkas** [electron/spectron#180]
I am initializing spectron like this:

```
const electron = path.resolve(__dirname, `../../../node_modules/.bin/electron-prebuilt-compile${process.platform === 'win32' ? '.cmd' : ''}`)

const paths = {
  root: path.join(__dirname, '../../../') || '.',
  main: path.join(__dirname, '../../../src/main/index.js')
}

const app = new Application({ path: electron, args: [paths.main] })
// app.start()
```

which worked fine with spectron `3.6.0`. Upgrading to `3.6.1` however causes an error message that has *no impact on the tests* (they still pass):

<img width="427" alt="bildschirmfoto 2017-04-11 um 15 05 00" src="https://cloud.githubusercontent.com/assets/3372410/24910499/45529c88-1ec8-11e7-9c27-53749a22f5f0.png">

I downgraded to 3.6.0 and the error disappeared.

**Comment by karacas** [comment]
same here, worked fine with spectron 3.6.0, get same error with spectron 3.6.1

**Comment by neighborhood999** [comment]
In `3.6.1`, I got this error:

```
[Error: Client initialization failed after 10 attempts: RuntimeError Client initialization failed after 10 attempts: ]
```

But in `3.6.0` not throw any error.

**Comment by kevinsawicki** [comment]
Thanks for reporting this, this should be fixed in 3.6.2
